examples/practice-typescript/src/components/TableHeader /HeaderItem.tsx

```diff
@@ -0,0 +1,22 @@
+interface HeaderItemProps {
```

Comment:
Recheck the directory name has extra spacing there

Reply:
Fixed